src/main/java/org/jenkinsci/plugins/ghprb/GhprbPullRequest.java

```diff
@@ -255,13 +255,13 @@ public boolean isWhiteListedTargetBranch() {
 
     private boolean isUpdated(GHPullRequest pr) {
         synchronized (this) {
-            Date lastUpdated = new Date();
+            
             boolean ret = false;
             try {
-                lastUpdated = pr.getUpdatedAt();
-                ret = updated.compareTo(lastUpdated) < 0;
+                Date lastUpdated = pr.getUpdatedAt();
+                ret = updated == null || updated.compareTo(lastUpdated) < 0;
```

Comment:
The reason I [wrapped this in a null check](https://github.com/jenkinsci/ghprb-plugin/pull/287/files#diff-70706d3124e337b7f717ad901c08d53aR261) for lastUpdated in #287 was to prevent a situation where `updated` is null and then `getUpdatedAt()` keeps returning null for the PR.

It's probably an unlikely scenario (possibly not even possible for getUpdatedAt() to return null without throwing an exception but I didn't look at the implementation in the github-api to verify).  But in the even that it does happen and the date kept remaining null, it would mark the PR as being updated and continually rebuild it.

Just an FYI on an unlikely but possible situation.